telemetry.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+package openmind;
+
+option csharp_namespace = "OpenMind";
+
+service TelemetryService {
+  /**
+   * TODO (BNR): We need endpoints for:
+   *             1. Version number - should respond with the server version
+   *             2. Supported devices - should respond with "Medtronic Summit RC+S"
+   *             3. Inspect Repository - should respond with the cached bridges/devices
+   */
+
+  /**
+   * List the version number for the Summit Server
+   */
+  rpc VersionNumber (VersionNumberRequest) returns (VersionNumberResponse); 
+  /**
+  * List supported devices.
+  */
+  rpc SupportedDevices (SupportedDevicesRequest) returns (SupportedDevicesResponse);
+  /**
+   * Inspect Repository.
+   */
+  rpc InspectRepository (InspectRepositoryRequest) returns (InspectRepositoryResponse);
+}
+
+message VersionNumberRequest {
+  /**
+   * Scheme-less URI describing the bridge resource.
+   * For example, '//summit/bridge/xxxxxxxxx' where
+   * 'xxxxxxxxx' is the serial number of the bridge.
+   */
+  string name = 1;
+}
+
+message VersionNumberResponse {
+  
+  /**
+   * Scheme-less URI describing the device resource. For example,
+   * '//summit/bridge/xxxxxx/device/yyyyyy' where 'xxxxxx' is the
+   * serial number of the bridge and 'yyyyyy' is the serial number
+   * of the device.
+   */
+  string name = 1;
```

Comment:
Same deal here, you don't need the name in the response because the version number is global